src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -190,6 +191,7 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
                                 Slots.RESPONSE_INDEX, Integer.toString(responseIdx),
                                 Slots.OPTION_INDEX, Integer.toString(i),
                                 Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                                Slots.MARGIN_LEFT, "style=\"margin-left:auto\"",
```

Comment:
alignment of input fields were already taken care, as size of column for label field is fixed. So, it is not required for `distributeToRecipients`.